app/controllers/api/order_cycles_controller.rb

```diff
@@ -1,10 +1,14 @@
 module Api
   class OrderCyclesController < Api::BaseController
     include EnterprisesHelper
-    respond_to :json
```

Comment:
not needed anymore?

Reply:
Nope, this is already declared in `Api::BaseController`.